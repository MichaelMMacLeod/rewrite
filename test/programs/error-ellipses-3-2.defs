(def (flatten (list (list (list $x ..) ..) ..))
  (list $x .. ..))