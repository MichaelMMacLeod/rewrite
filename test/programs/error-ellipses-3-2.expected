error[E002]: too few ellipses
./test/programs/error-ellipses-3-2.defs:1:33
   |
 1 | (def (flatten (list (list (list $x ..) ..) ..))
   |                                 ^^ matched with 3 ellipses in pattern
./test/programs/error-ellipses-3-2.defs:2:9
   |
 2 |   (list $x .. ..))
   |         ^^ used with 2 ellipses in constructor
help: variables must be used with the same number of ellipses they were matched with
